e2e/cypress/integration/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const iamVersion = <string><Object>Cypress.env('IAM_VERSION') || 'v2.0';
 export const describeIfIAMV2 = iamVersion.match(/v2/) ? describe : describe.skip;
 export const describeIfIAMV2p1 = iamVersion === 'v2.1' ? describe : describe.skip;
+export const runFlaky = <boolean><Object>Cypress.env('RUN_FLAKY') || false;
```

Comment:
Can you explain why we need `<boolean><Object>`? 🤔

Reply:
Looks like we don't!